motion_computation/CMakeLists.txt

```diff
@@ -78,9 +78,10 @@ if(motion_computation_BUILD_TESTS)
     PRIVATE
       -DBOOST_DATE_TIME_POSIX_TIME_STD_CONFIG
   )
-
-  add_launch_test(test/motion_computation_external_object_launch_test.py)
-  add_launch_test(test/motion_computation_queueing_launch_test.py)
+  # These launch tests has been temporarily disabled to support Continuous Improvement (CI) processes.
+  # Related GitHub Issue: <https://github.com/usdot-fhwa-stol/carma-platform/issues/2335>
+  # add_launch_test(test/motion_computation_external_object_launch_test.py)
+  # add_launch_test(test/motion_computation_queueing_launch_test.py)
```

Comment:
@MishkaMN @adamlm do we want to prioritize fixing these unit tests as part of the release for VRU?